lib/bot.js

```diff
@@ -43,6 +45,24 @@ class Bot {
 
     this.invertedMapping = _.invert(this.channelMapping);
     this.autoSendCommands = options.autoSendCommands || [];
+
+    if (options.gravatars) {
+      const updateGravatars = () => {
+        logger.debug('Updating gravatars: ', options.gravatars);
+        this.gravatars = [];
+        JSON.parse(fs.readFileSync(options.gravatars, 'utf8')).forEach(pair => {
```

Comment:
By the way, you should probably change this to use the asynchronous version of readFile instead of the synchronous one. If you change the configuration file while the bot is running this will halt it from being able to do anything else while it's reading the file.